app/models/decidim/ludens/participative_action.rb

```diff
@@ -9,8 +9,6 @@ class ParticipativeAction < ApplicationRecord
                  foreign_key: "decidim_organization_id",
                  class_name: "Decidim::Organization"
 
-      validates :organization, presence: true
```

Comment:
Are you sure you want to delete the validation?

Reply:
Nope it's an error i'm reverting that

Reply:
Actually it's not a mistake, it's a correction by rubocop : it's already defined by the belongs_to attribute so it's redondant to add the validation

Reply:
Ok !